### PR TITLE
Gold standard ds

### PR DESCRIPTION
**Pull request by jarsba**
- Add `leaves_segmented_master.csv` that contains label columns
- Add a golden plant dataset `plant_data_golden.csv` that contains only plant images larger than 50 000 square pixels or images roughly larger than 225 x 225 pixels. Dataset also has two new columns, `Area` and `N of SIFT features`

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from data_exploration import *
+from dataloaders import *
+from models import *
+from preprocessing import *
+from segmentation import *
+from utils import *
```
Comment (Redande) [inline __init__.py:6]: missing new line

File: preprocessing/preprocess_leaf_data.py [hunk 2]
```diff
@@ -75,7 +75,26 @@
...
+
 # %%
-file_name = "leaves_segmented.csv"
+
+file_name = "leaves_segmented_master.csv"
 file_path = os.path.join(DATA_FOLDER, file_name)
-leaves_segmented.to_csv(file_path)
+leaves_segmented.to_csv(file_path, index=False)
+
+# %%
```
Comment (Redande) [inline preprocessing/preprocess_leaf_data.py:100]: missing new line

**Review by Redande — APPROVED** [review]
Looks good! A couple of missing newlines at end of files, but nothing critical and I think it could be merged to main already as it is.
VSCode has a setting under Preferences > Settings > search for "insert final newline". When enabled, VSCode adds the newline on file save if it's missing.

File: preprocessing/preprocess_golden_datasets.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  print(f"Image with {n_features} features and area {area}")
+  
+  image = cv2.imread(image_path)
+  
+  plt.imshow(image)
+  plt.show()
+
+# %%
+
+# All images are so similar that no need for golden dataset
```
Comment (Jakoviz) [inline preprocessing/preprocess_golden_datasets.py:148]: Splitting the code into a couple of functions would help readability.

**Review by Jakoviz — APPROVED** [review]
Looks good. My only comment is that preprocess_golden_datasets could be refactored a bit more readable using functions.

**Pull request merged**